app/src/main/java/io/novafoundation/nova/app/root/presentation/splitScreen/SplitScreenViewModel.kt

```diff
@@ -72,4 +78,18 @@ class SplitScreenViewModel(
             null
         )
     }
+
+    fun onNavigationAttached() {
+        consumablePayload.use {
```

Comment:
I suggest to rename it to `useOnce` and rename the `Consumer` class to `UseOnce` or something similar